specification/metadata/metadata.mdpp

```diff
@@ -0,0 +1,13 @@
+# Metadata
+
+The FOCUS specification defines a metadata structure that is to be supplied by data providers to facilitate practioners in data ingestion, data locations, schema of the FOCUS dataset.
+
+!INCLUDE "metadata_structure.md",1
+!INCLUDE "metadata_provider.md",1
```

Comment:
Nesting with mdpp will be a lot more harder if we want to manage subsections. I'd like to see if we can keep it more flat - it would be consistent with what we did with columns (where we previously had metrics and dimensions).

Reply:
I am working on a proposal to rework the layout of this content. Hoping to have it available by tomorrow.